tests/plotting/bokeh/testgraphplot.py

```diff
@@ -71,7 +71,7 @@ def test_graph_inspection_policy_edges(self):
         renderer = plot.handles['glyph_renderer']
         hover = plot.handles['hover']
         self.assertIsInstance(renderer.inspection_policy, EdgesAndLinkedNodes)
-        self.assertEqual(hover.tooltips, [('start', '@{start}'), ('end', '@{end}')])
+        self.assertEqual(hover.tooltips, [('start', '@{start_values}'), ('end', '@{end_values}')])
```

Comment:
My understanding is these names are a convention established by holoviews for use in bokeh's hover tools. It would be good either to document this in this PR or open an issue as a reminder to document these names.